File: Java/Lista 3/L3_Ex1.4/src/UsuarioGuest.java

public class UsuarioGuest extends Usuario {
    
    public UsuarioGuest(String login) {
        super(login);
    }
    
    public void descricaoGuest(){
        System.out.println("O usuario Guest so pode ler arquivos");
    }
    
    public void despedidaGuest(){
        System.out.println("Até logo" + this.getLogin() + "!");
    }
}

File: Swift/Lista_04_EX04/UsuarioGuest.swift
import Foundation
 
 public class UsuarioGuest : Usuario{
 
	override func descricaoGuest(){
		print("O usuario Guest so pode ler arquivos. ")
	}
	
	func despedidaGuest() {
		print("Até logo \(login)! ")
	}
	
}